### DISCUSSION TITLE
Please allow option to set/change the default time for appearing of the upper control panel when connecting in fullscreen mode ... (longer cm timeout)

### DISCUSSION THREAD

**Discussion by givbra** [rustdesk/rustdesk#3894] [Ideas]
Connecting remotely to a PC in a fullscreen mode it is quite annoying when trying to close some window on the remote machine and if I wait some seconds more with the mouse on the right upper corner of the screen I accidentally very often close the whole remote session of RustDesk and have to log in again ... Please make the default timeout for appearing the upper control panel bar twice or more (or make an option for users to adjust it to their prefered time interval) longer to avoid closing session by mistake ... It is good that the default timeout for appearing of the upper control panel is a little longer of the one in Anydesk, but still too quick for normal work without problems ... After all working with this panel is not a priority, when you are in fullscreen mode - you really don't need it to appear often and by mistake ... Better to wait more to appear when in need than to constantly closing session by mistake ...

**Comment by truthsword** [comment]
Came here to say this. Would like to delay fullscreen menu dropdown 2-3 seconds. Or disable it in favor of Fxx key toggle.
